common/src/main/scala/monix/connect/common/Operators.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2014-2020 by The Monix Project Developers.
```

Comment:
You might want to say "Monix Connect". And link to the project's repository, until we'll settle for a documentation website.

As I said, we could do some arrangement with `monix.io`, maybe have a `monix.io/connect/` or a `connect.monix.io`, but until then let's link to the repository, such that the content is related to Connect.